CODEOWNERS

```diff
@@ -84,4 +84,6 @@ android/ @ahumesky @jin
 /src/main/java/com/google/devtools/build/lib/syntax/ @brandjon @c-parsons @dkelmer @dslomov @laurentlb @vladmos
 
 /tools/coverage/ @iirina
```

Comment:
anything in tools/coverage will fallthrough to this line, so you'll want to remove that as well

Reply:
But that's different that tools/test. I'm OK with owning what's under tools/coverage, or did I miss something?

Reply:
ah OK, d'oh - I parsed "Make @iirina codeowners of only the coverage parts of tools/test." as you not being the owner of tools/coverage - sorry for the noise! Monday mornings :-)